.github/actions/install-deps/action.yml

```diff
@@ -52,8 +52,8 @@ runs:
         }
         sudo mkdir -p "${{ inputs.verilator-path }}"
         sudo chmod 777 "${{ inputs.verilator-path }}"
-        tar -C "${{ inputs.verilator-path }}" -xvzf "/tmp/${VERILATOR_TAR}" --strip-components=1
-        echo "${{ inputs.verilator-path }}/bin" >> "$GITHUB_PATH"
+        tar -C "${{ inputs.verilator-path }}" -xvzf "/tmp/${VERILATOR_TAR}"
```

Comment:
This is needed because when we build verilator we set prefix to include its version.

Reply:
Oh, I see. I had assumed they'd be using relative paths within the install directory.